src/themes/Default.jsx

```diff
@@ -270,39 +271,43 @@ export const config = {
     checkbox: {
       borderColor: colors.darkGray3,
       color: colors.darkGray2,
+      disabledOpacity: 0.6,
+
       sm: {
         height: '2.2rem',
         size: '1.4rem',
         top: '0.7rem',
         innerSize: '1.1rem',
-        paddingLeft: '2rem'
+        paddingLeft: '1rem'
```

Comment:
These padding values don't seem right anymore, the checkbox is overlapping with the label.